File: CountdownTimer.js
//A Javascript version of a countdown timer so i can see the differences in how different languages approach the same exercises.

let time = 5;
let countdown = time * 60;
let prompt = false;

let countdownTimer = setInterval(function(){
  if (countdown === 0) {
    clearInterval();
    if (!prompt) {
    console.log("Countdown finished!");
     prompt = true;
    return;
    }
  } else {
    countdown -= 1;
    let minutes = Math.floor(countdown / 60);
    let seconds = countdown % 60;
  
    minutes = minutes < 10 ? "0" + minutes : minutes;
    seconds = seconds < 10 ? "0" + seconds : seconds;
    let x = `${minutes}: ${seconds}`;
    console.log(x);
    }
  }, 1000);

File: CountdownTimer.cs
// A very simple countdown timer to test my working knowledge and skills learnt so far in C#

CountdownTimer(5);
		
		void CountdownTimer(int timer)
		{
			int countdown = timer * 60;
		
			if (countdown > 0)
			{
				do
				{
				countdown -= 1;
				int minutes = countdown / 60;
				int seconds = countdown % 60;
					if (seconds < 10)
					{
						Console.WriteLine($"Countdown: {minutes}:0{seconds}");
					}
					else
					{
						Console.WriteLine($"Countdown: {minutes}:{seconds}");
					}
				} while (countdown > 0);
			}
			Console.WriteLine("Countdown timer finished!");
		} 
